restcomm/restcomm.mgcp/pom.xml

```diff
@@ -73,6 +73,12 @@
 			<scope>provided</scope>
 		</dependency>
 
+	  <dependency>
```

Comment:
@gvagenas do you mean apache commons? 
we are using OctedString class from this lib (`snmp4j `) on mediaserver too. need to change on both sides